src/client/components/App.js

```diff
@@ -21,7 +21,7 @@ const onboardingSupplier = {
   registrationNumber: "MI342323"
 };
 const username = 'john.doe@ncc.com';
-const actionUrl = 'http://localhost:3001'
+const actionUrl = 'http://localhost:8080'
```

Comment:
This is a test UI (used only in dev and nowhere else) so I would like to keep it simple.